stroyprombeton/context/catalog.py

```diff
@@ -6,44 +6,16 @@
 This using will becom possible after se#567 released.
 """
 
-from functools import partial, reduce
-from operator import or_
+from functools import partial
 
 from django.conf import settings
-from django.db import models
-from django.shortcuts import get_object_or_404, _get_queryset
+from django.shortcuts import get_object_or_404
 
 from catalog import context, typing
 from images.models import Image
 from pages import context as pages_context
-from search.search import QuerySetType
 from stroyprombeton import models as stb_models, request_data
-
-
-# doubled `search.search.search` here to change name to product__name.
-# Redesign search module on refarm side in more extensible way.
-def search(term: str, model_type: typing.Union[models.Model, models.Manager, QuerySetType],
-           lookups: list, ordering) -> QuerySetType:
-    """Return search results based on a given model."""
-    def _get_Q(lookup):
-        return models.Q(**{lookup: term})
-
-    term = term.strip()
-    query_set = _get_queryset(model_type)
-    query = reduce(or_, map(_get_Q, lookups))
-
-    return (
-        query_set.filter(query)
-        .annotate(
-            is_name_start_by_term=models.Case(models.When(
-                product__name__istartswith=term,
-                then=models.Value(True)),
-                default=models.Value(False),
-                output_field=models.BooleanField()
-            )
-        )
-        .order_by(models.F('is_name_start_by_term').desc(), *ordering)
-    )
+from . import options
```

Comment:
Let's use relative imports only in `__init__` files to reduce path to nested modules. We have convention for this and [pep8](https://www.python.org/dev/peps/pep-0008/#imports) recommends this approach